bin/standalone-installer-windows

```diff
@@ -0,0 +1,132 @@
+#
```

Comment:
I think it belies that you could run it on Unix under pwsh, when in fact you can't. Plus, the very next line already says it's a PowerShell program:

https://github.com/nextstrain/cli/blob/40724d70fee962f43bfa5c3a02eff1f134701306/bin/standalone-installer-windows#L1-L2

Reply:
If this is more about editor config than human's reading, I did put a vim modeline at the bottom:

https://github.com/nextstrain/cli/blob/40724d70fee962f43bfa5c3a02eff1f134701306/bin/standalone-installer-windows#L132

but this will only work with compatible modeline-supporting editors. We could switch to a shebang if that's better, I suppose.